### PR TITLE
Instance order  - do not merge

### PR DESCRIPTION
**Pull request by JeffreyDavidsz**
Start of instance ordering, DOES NOT WORK YET

### CODE REVIEW

File: lib/instance.js [hunk 1]
```diff
@@ -563,8 +564,9 @@ instance.prototype.connect = function (client) {
 	});
 
 	client.on('instance_order', function(old_index, new_index) {
-		var instances = self.instances;
+		var instances;
 		if (instances !== undefined) {
```
Comment (haakonnessjoen) [inline lib/instance.js:568]: This if will never go through, since you just defined instances as undefined :P

Comment (krocheck) [reply lib/instance.js:568]: Funny ... I was just looking through this code last night.  Doing this ends up being more difficult than I expected.  The drag order is using a numeric index for the sort order but ultimately the keys in the array its sorting are the instance IDs.  I haven't found an elegant way to make that all work yet.

Comment (JeffreyDavidsz) [reply lib/instance.js:568]: the same reason why my time shifted to something else ;-)

Comment (krocheck) [reply lib/instance.js:568]: I'm going with punt to v3

**Pull request closed without merging**